### PR TITLE
[S2GRAPH-11] remove JsNumber.as[Double] on sortBy in PostProcess.

### PR DESCRIPTION
**Pull request by SteamShon**
remove JsNumber.as[Double] on sortBy in PostProcess.

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -5,6 +5,8 @@ Release 0.12.1 - unreleased
   NEW FEATURES
 
   IMPROVEMENT
+ 
+    S2GRAPH-11: Performance tuning on sort by scoreSum on PostProcess
```
Comment (hsleep) [inline CHANGES]: I think that CHANGES file should not be changed in this PR.

**Comment by SteamShon** [comment]
I merged changed master into this PR. please review this again

**Comment by elric-k** [comment]
nice work. 
+1

**Comment by hsleep** [comment]
+1

**Pull request closed without merging**